server/autotest_server/run_test_script.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import msgspec
+
+# Different parts of data that we need is sent on stdin separated by newlines
+tester_type = sys.stdin.readline().strip()
+resource_settings = msgspec.json.decode(sys.stdin.readline().strip())
+specs_json = sys.stdin.read()
+
+# Dynamically import the models module based on the tester type
+models_module = __import__("testers.models", fromlist=[f"{tester_type.capitalize()}TestDatum"])
```

Comment:
Yeah, I'd rather you use `importlib` here to be consistent with other parts of the code. Also I don't think the `fromlist` is necessary if you're using `getattr` below